pkg/apps/utils.jsx

```diff
@@ -57,14 +57,14 @@ export function icon_url(path_or_url) {
 
 export const ProgressBar = ({ title, data }) => {
     if (data.waiting) {
-        return (
-            <div>
-                <div className="pull-right spinner spinner-sm" />
-                <div className="progress-title">
-                    {_("Waiting for other programs to finish using the package manager...")}
-                </div>
-            </div>
-        );
+        return (<Split>
```

Comment:
I would deserve some padding but not a big deal.
![Screen Shot 2020-12-07 at 11 28 32](https://user-images.githubusercontent.com/14921356/101339976-687da980-387f-11eb-8ae2-be369c6d7570.png)